cvat-ui/src/components/cvat-app.tsx

```diff
@@ -496,6 +497,16 @@ class CVATApplication extends React.PureComponent<CVATAppProps & RouteComponentP
                             path='/auth/login-with-social-app/'
                             component={LoginWithSocialAppComponent}
                         />
+                        <Route
+                            exact
+                            path='/auth/login-with-oidc/'
+                            component={LoginWithSocialAppComponent}
```

Comment:
I think we can use `path={["/users/:id", "/profile/:id"]}` an array of paths instead